### PR TITLE
Adds manifest key so that we use the same extension ID

### PR DESCRIPTION
**Pull request by NejcZdovc**

### CODE REVIEW

File: app/manifest.dev.json [hunk 1]
```diff
@@ -22,5 +22,6 @@
     "newtab": "newTabPage.html"
   },
   "permissions": [ "contentSettings", "management", "tabs", "storage", "webNavigation" ],
-  "content_security_policy": "default-src 'self'; script-src 'self' http://localhost:3000 https://localhost:3000 chrome-extension://naelnlokfbfcbonlpmkmbdkjogkpkigm 'unsafe-eval'; connect-src http://localhost:3000 https://localhost:3000; style-src * 'unsafe-inline' 'self' blob:; img-src 'self' data:;"
+  "content_security_policy": "default-src 'self'; script-src 'self' http://localhost:3000 https://localhost:3000 chrome-extension://naelnlokfbfcbonlpmkmbdkjogkpkigm 'unsafe-eval'; connect-src http://localhost:3000 https://localhost:3000; style-src * 'unsafe-inline' 'self' blob:; img-src 'self' data:;",
+  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAupOLMy5Fd4dCSOtjcApsAQOnuBdTs+OvBVt/3P93noIrf068x0xXkvxbn+fpigcqfNamiJ5CjGyfx9zAIs7zcHwbxjOw0Uih4SllfgtK+svNTeE0r5atMWE0xR489BvsqNuPSxYJUmW28JqhaSZ4SabYrRx114KcU6ko7hkjyPkjQa3P+chStJjIKYgu5tWBiMJp5QVLelKoM+xkY6S7efvJ8AfajxCViLGyDQPDviGr2D0VvIBob0D1ZmAoTvYOWafcNCaqaejPDybFtuLFX3pZBqfyOCyyzGhucyCmfBXJALKbhjRAqN5glNsUmGhhPK87TuGATQfVuZtenMvXMQIDAQAB"
```
Comment (bbondy) [inline app/manifest.dev.json]: This also needs to be added to the prod manifest file pls

Comment (NejcZdovc) [reply app/manifest.dev.json]: done

**Comment by bsclifton** [comment]
This is interesting- any reason in particular that we'd want to use the same extension ID as browser-laptop?

**Comment by bbondy** [comment]
no reason why the ID had to be the same, a key was needed so that all developers shared the same ID though.

**Pull request merged**